### ISSUE TITLE
can not apply migration 

### ISSUE THREAD

**Issue by bender92** [Netflix/dispatch#1331]
**Describe the bug**
 command dispatch database upgrade doesn't work, how to apply migrations? 

_dispatch database init_ is success 
_dispatch database upgrade_ hang out with logs

```
/events/slack/action                                         False            POST
/events/slack/menu                                           False            POST:/usr/local/lib/python3.8/site-packages/dispatch/main.py:<module>:176
INFO:Migrating dispatch core schema...:/usr/local/lib/python3.8/site-packages/dispatch/database/revisions/core/env.py:run_migrations_online:51
INFO:Context impl PostgresqlImpl.:/usr/local/lib/python3.8/site-packages/alembic/runtime/migration.py:__init__:164
INFO:Will assume transactional DDL.:/usr/local/lib/python3.8/site-packages/alembic/runtime/migration.py:__init__:167
```

**Comment by kevgliss** [comment]
I believe this has been fixed in #1358. Essentially, there was a trigger that was locking the table during the migration.

**Comment by bender92** [comment]
yes fixed
